modules/seqexec/model/src/main/scala/seqexec/model/NodAndShuffleStatus.scala

```diff
@@ -10,16 +10,24 @@ import seqexec.model.GmosParameters._
 import monocle.macros.Lenses
 import squants.Time
 
+final case class NSRunningState(action: NSAction, sub: NSSubexposure)
+
+object NSRunningState {
+  implicit val equalNSRunningState: Eq[NSRunningState] =
+    Eq.by(x => (x.action, x.sub))
+}
+
 @Lenses
 final case class NodAndShuffleStatus(
   observing: ActionStatus,
   totalExposureTime: Time,
   nodExposureTime: Time,
-  cycles: NsCycles
+  cycles: NsCycles,
+  state: Option[NSRunningState]
```

Comment:
@rpiaggio This will be populated once the sequence starts running indicating the current subexposure and the state